src/Symfony/Component/Yaml/Parser.php

```diff
@@ -634,6 +643,11 @@ private function isCurrentLineComment()
         return '' !== $ltrimmedLine && $ltrimmedLine[0] === '#';
     }
 
+    private function isCurrentLineLastLineInDocument()
```

Comment:
Not sure, we don't pass any argument so it's not really clear to what the method name refers to imo. Note that the `isCurrentLine` prefix is a common pattern in the parser (e.g. `isCurrentLineComment()`, `isCurrentLineBlank()` and so on).